lib/s3-html/log.html

```diff
@@ -478,7 +478,7 @@
     content += await fetch_from(filename, bytes);
     set_content(content);
 
-    console.log('Thank you for using s3-streamer.  Have a nice day.');
+    console.log('Thank you for using s3streamer.  Have a nice day.');
```

Comment:
lolwat

On account of that's the name of the library? :)

Reply:
Yeah, "s3-streamer" as such isn't a thing any more.